internal/db/schema/migrations/oss/postgres/84/05_alias_view.up.sql

```diff
@@ -0,0 +1,15 @@
+-- Copyright (c) HashiCorp, Inc.
+-- SPDX-License-Identifier: BUSL-1.1
+
+begin;
+
+  -- alias_view defines a view that retrieves the common columns from the
+  -- subtype alias tables.
+  create view alias_view as
```

Comment:
I think we tend to call this something like `_all_subtypes`, ie `alias_all_subtypes`?